modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2045,6 +2046,17 @@ private Expression<?> _getFunctionExpression(
 		throw new IllegalArgumentException("Invalid function " + function);
 	}
 
+	private Iterable<Object> _getIterable(Object value) {
+		if (value instanceof Iterable) {
+			return (Iterable)value;
+		}
+
```

Comment:
@devpedrotavares in which scenario will this condition be true?

Reply:
This is the standard scenario. When the user submits an entry via the UI, it is parsed to an `ArrayList` (which is also an `Iterable`). As well as when the entry is submitted through the API as `["key1", "key2"]` for example.

I only chose `Iterable` because it's more generic and before we we're treating `JSONArray` also. But I guess we could switch to `List`.